src/main/java/frc/robot/config/TunerConstants.java

```diff
@@ -86,32 +86,32 @@ public class TunerConstants {
   private static final int kFrontLeftSteerMotorId = 2;
   private static final int kFrontLeftEncoderId = 1;
 
-  private static final double kFrontLeftXPosInches = 12;
-  private static final double kFrontLeftYPosInches = 12;
+  private static final double kFrontLeftXPosInches = 12.375;
```

Comment:
This puts the wheels outside the robot perimeter, doesn't it? 

0.61 meters (from pathplanner settings.json) is 24.0157 inches. 
12.375 inches * 2 = 24.75 inches.

Reply:
We'll be leaving them as 12 for waco but we can measure and adjust them after

Reply:
i became malachi again, this was ryan